verify.css

```diff
@@ -21,17 +21,15 @@ a {
     color: #7289da;
 }
 
-ul form details {
+ul form {
```

Comment:
The alterations for this section were straight from the site; feel free to alter if needed.